2025-01-11 15:43:12,288:DEBUG:certbot.main:certbot version: 0.31.0
2025-01-11 15:43:12,289:DEBUG:certbot.main:Arguments: ['--manual', '--preferred-challenges', 'http', '-d', 'htdguide.com']
2025-01-11 15:43:12,289:DEBUG:certbot.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2025-01-11 15:43:12,393:DEBUG:certbot.log:Root logging level set at 20
2025-01-11 15:43:12,393:INFO:certbot.log:Saving debug log to /var/log/letsencrypt/letsencrypt.log
2025-01-11 15:43:12,397:DEBUG:certbot.plugins.selection:Requested authenticator manual and installer None
2025-01-11 15:43:12,397:DEBUG:certbot.plugins.selection:Single candidate plugin: * manual
Description: Manual configuration or run your own shell scripts
Interfaces: IAuthenticator, IPlugin
Entry point: manual = certbot.plugins.manual:Authenticator
Initialized: <certbot.plugins.manual.Authenticator object at 0x7f5ef2e9eeb8>
Prep: True
2025-01-11 15:43:12,398:DEBUG:certbot.plugins.selection:Selected authenticator <certbot.plugins.manual.Authenticator object at 0x7f5ef2e9eeb8> and installer None
2025-01-11 15:43:12,398:INFO:certbot.plugins.selection:Plugins selected: Authenticator manual, Installer None
2025-01-11 15:43:12,406:DEBUG:certbot.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/2164207815', new_authzr_uri=None, terms_of_service=None), 7dfca5f71bc05a3fc4f8415e86b31071, Meta(creation_dt=datetime.datetime(2025, 1, 11, 15, 35, 39, tzinfo=<UTC>), creation_host='38a289705d13'))>
2025-01-11 15:43:12,407:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2025-01-11 15:43:12,408:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2025-01-11 15:43:12,970:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 746
2025-01-11 15:43:12,972:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sat, 11 Jan 2025 15:43:12 GMT
Content-Type: application/json
Content-Length: 746
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "B3RZt06cgOI": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2025-01-11 15:43:12,975:INFO:certbot.main:Obtaining a new certificate
2025-01-11 15:43:13,049:DEBUG:certbot.crypto_util:Generating key (2048 bits): /etc/letsencrypt/keys/0001_key-certbot.pem
2025-01-11 15:43:13,052:DEBUG:certbot.crypto_util:Creating CSR: /etc/letsencrypt/csr/0001_csr-certbot.pem
2025-01-11 15:43:13,052:DEBUG:acme.client:Requesting fresh nonce
2025-01-11 15:43:13,053:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2025-01-11 15:43:13,214:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2025-01-11 15:43:13,216:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sat, 11 Jan 2025 15:43:13 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: yoRvt02Q15qZ-W-xuRma3w5CkLb2E0T5oNpMiAs-63HO_HGtALU
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2025-01-11 15:43:13,217:DEBUG:acme.client:Storing nonce: yoRvt02Q15qZ-W-xuRma3w5CkLb2E0T5oNpMiAs-63HO_HGtALU
2025-01-11 15:43:13,218:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "htdguide.com"\n    }\n  ]\n}'
2025-01-11 15:43:13,220:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjE2NDIwNzgxNSIsICJub25jZSI6ICJ5b1J2dDAyUTE1cVotVy14dVJtYTN3NUNrTGIyRTBUNW9OcE1pQXMtNjNIT19IR3RBTFUiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "nOA9FionQaWSJLLXqJeR8xBIsvC_VWM61LuZJuy2clpk9y9amcWpcYsX6I4ftV1basWEmKVKr3M1s2DlcCNf-fDkx-IyEJho3nLmpWrtWgRcke1-Rvi-vaNou52ivId0fkqFVQfT7wp75L3o2mFj2qlgK7nqNN6k0E7z2tjZgLBhHRCu-J2RmrUETNVWMoUKt77Tp3I605cbDypnomBpUK4Z4dnrFHpyDpWOZ-HFMbkkdtwbglJelXP5Xe7zekACeIbH7BE72NEGyiW7gFpGPw5Kfa6tQUbrcPkPfShHm0CDSfKwFgnzC1O2brYslne4dL7JYHM1lnqH478FWk1kdg",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogImh0ZGd1aWRlLmNvbSIKICAgIH0KICBdCn0"
}
2025-01-11 15:43:13,517:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 346
2025-01-11 15:43:13,518:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Sat, 11 Jan 2025 15:43:13 GMT
Content-Type: application/json
Content-Length: 346
Connection: keep-alive
Boulder-Requester: 2164207815
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/2164207815/343283407525
Replay-Nonce: yoRvt02Qr9BsHMlHzunlj81-LuFTLdpyJNlQQQ4NRDX5XSeXqMc
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2025-01-18T15:43:13Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "htdguide.com"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz/2164207815/459109793735"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/2164207815/343283407525"
}
2025-01-11 15:43:13,519:DEBUG:acme.client:Storing nonce: yoRvt02Qr9BsHMlHzunlj81-LuFTLdpyJNlQQQ4NRDX5XSeXqMc
2025-01-11 15:43:13,519:DEBUG:acme.client:JWS payload:
b''
2025-01-11 15:43:13,521:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2164207815/459109793735:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjE2NDIwNzgxNSIsICJub25jZSI6ICJ5b1J2dDAyUXI5QnNITWxIenVubGo4MS1MdUZUTGRweUpObFFRUTROUkRYNVhTZVhxTWMiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIxNjQyMDc4MTUvNDU5MTA5NzkzNzM1In0",
  "signature": "OMCydRjYQ7SfCf0nONcwBeLMbrtrjYDDqvAjWU0NAW7BaYlyvzMRZZhIGjcuB0JzSevDiOmUgLmg4ofREmmECCNAQjRuM7SkMXp8WalwLLW7NxChzL35Ab1zdxH8r1KiYdHuIWXYJE5Z7_CUdBFIi-kh4ybHULlDSTBBVPb3YAYRUP9qar3QVisC95kJSmb5xDIzEUeJ8eN91roN5Sz5r_oI5h0wmrfCWY0pWDz3IY0DSVeDRjWLgGyDft-C4NgqKsDR5eCiOkBZl-WTFm1npvbSCbhFny-2DASAwQj_Mub870ul5-FBQqHUXTmfzAWndh0XIVP3MV-WrJX7e-6o2g",
  "payload": ""
}
2025-01-11 15:43:13,704:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2164207815/459109793735 HTTP/1.1" 200 820
2025-01-11 15:43:13,705:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sat, 11 Jan 2025 15:43:13 GMT
Content-Type: application/json
Content-Length: 820
Connection: keep-alive
Boulder-Requester: 2164207815
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: yoRvt02Q6xg35aMHRRoPLoGrEx_kg0l6-zLvQjfNWTiTZu6EhtY
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "htdguide.com"
  },
  "status": "pending",
  "expires": "2025-01-18T15:43:13Z",
  "challenges": [
    {
      "type": "dns-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2164207815/459109793735/0Y1D8Q",
      "status": "pending",
      "token": "D9OXacffwIfCwbLeiK5IFrFuBJZfbJ2eCDWKljAmN5M"
    },
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2164207815/459109793735/xYPqnA",
      "status": "pending",
      "token": "D9OXacffwIfCwbLeiK5IFrFuBJZfbJ2eCDWKljAmN5M"
    },
    {
      "type": "tls-alpn-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2164207815/459109793735/hMgCKA",
      "status": "pending",
      "token": "D9OXacffwIfCwbLeiK5IFrFuBJZfbJ2eCDWKljAmN5M"
    }
  ]
}
2025-01-11 15:43:13,706:DEBUG:acme.client:Storing nonce: yoRvt02Q6xg35aMHRRoPLoGrEx_kg0l6-zLvQjfNWTiTZu6EhtY
2025-01-11 15:43:13,707:INFO:certbot.auth_handler:Performing the following challenges:
2025-01-11 15:43:13,708:INFO:certbot.auth_handler:http-01 challenge for htdguide.com
2025-01-11 15:44:15,990:DEBUG:certbot.error_handler:Encountered signals: [1]
2025-01-11 15:44:15,991:DEBUG:certbot.error_handler:Calling registered functions
2025-01-11 15:44:15,991:INFO:certbot.auth_handler:Cleaning up challenges
2025-01-11 15:44:15,992:DEBUG:certbot.error_handler:Calling signal 1
2025-01-11 15:44:15,992:DEBUG:certbot.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/bin/certbot", line 11, in <module>
    load_entry_point('certbot==0.31.0', 'console_scripts', 'certbot')()
  File "/usr/lib/python3/dist-packages/certbot/main.py", line 1365, in main
    return config.func(config, plugins)
  File "/usr/lib/python3/dist-packages/certbot/main.py", line 1250, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/usr/lib/python3/dist-packages/certbot/main.py", line 121, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/usr/lib/python3/dist-packages/certbot/client.py", line 410, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/usr/lib/python3/dist-packages/certbot/client.py", line 353, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/usr/lib/python3/dist-packages/certbot/client.py", line 389, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, best_effort)
  File "/usr/lib/python3/dist-packages/certbot/auth_handler.py", line 85, in handle_authorizations
    self.verify_authzr_complete(aauthzrs)
  File "/usr/lib/python3/dist-packages/certbot/auth_handler.py", line 345, in verify_authzr_complete
    raise errors.AuthorizationError("Incomplete authorizations")
certbot.errors.AuthorizationError: Incomplete authorizations
2025-01-11 15:44:15,995:DEBUG:certbot.util:Exception occurred releasing lock: LockFile(/etc/letsencrypt/.certbot.lock) <released>
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/certbot/util.py", line 141, in _release_locks
    dir_lock.release()
  File "/usr/lib/python3/dist-packages/certbot/lock.py", line 122, in release
    compat.release_locked_file(self._fd, self._path)
  File "/usr/lib/python3/dist-packages/certbot/compat.py", line 156, in release_locked_file
    os.remove(path)
FileNotFoundError: [Errno 2] No such file or directory: '/etc/letsencrypt/.certbot.lock'
